source/hardware/controllers/native_instruments_traktor_kontrol_s2_mk3.rst

```diff
@@ -93,7 +97,7 @@ Mixxx mapping
 | (19) Quantize    | Toggles quantization for both decks                                                                                         |                                                     |
 | button           |                                                                                                                             |                                                     |
 +------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+
-| (20) Gain knob   | Adjusts the gain for the master output                                                                                      |                                                     |
+| (20) Gain knob   | Adjusts the hardware gain - it is unlinked to the software main gain since v2.3                                             |                                                     |
```

Comment:
This doesn't belong into the table. This is the manual for Mixxx 2.3 so there's no reason to document behavior for earlier mixxx versions.